pyconbalkan/timetable/views.py

```diff
@@ -1,16 +1,38 @@
+
+from django.shortcuts import render, redirect
 from rest_framework import viewsets
-from pyconbalkan.timetable.models import Timetable, Presentation, Room
+
+from pyconbalkan.conference.models import Conference
+from pyconbalkan.timetable.models import Room, Slot, Timetable
 from pyconbalkan.timetable.serializers import TimetableSerializer
-from django.shortcuts import render
 
 
 class TimetableViewSet(viewsets.ModelViewSet):
     queryset = Timetable.objects.all()
-    serializer_class = TimetableSerializer  
+    serializer_class = TimetableSerializer
 
 
 def timetable_view(request):
-    presentations = Presentation.objects.all()
+    conference = Conference.objects.filter(active=True)
+    if conference[0].timetable_pdf:
+        return redirect('/'+conference[0].timetable_pdf.name)
```

Comment:
So this should redirect to the absolute url:

https://docs.djangoproject.com/en/dev/topics/files/#using-files-in-models

>>> car.photo.url
'http://media.example.com/cars/chevy.jpg'